### PR TITLE
[Ancient Station] First Round Of Changes/Additions

### LINKED ISSUE #28947: Ancient Station Transit Tube
When entering the transit tube on the ancient station, to get into Delta (The station with research) from Charlie, you will be slammed with a massive amount of overpressure, enough to inflict injuries up to and including bleeding. It knocked me from yellow into orange, not bleeding into bleeding.

### PR DESCRIPTION
**Pull request by Steelpoint**
First main changes based on feedback and similar.

* Fixed minor mapping issues, such as broken glass and similar.
* Added some bluespace crystals to the Hivebot ship for RnD.
* All station helmets and the hardsuit helmet no longer block vision.
* NASA Engineering Void Suit now has a proper slowdown of 4.
* Two pairs of 'worn down insulated gloves' have been added to engineering. Think of them as Budget Gloves, but they have a much better chance of working.
* New movement sounds for the prototype RIG Hardsuit, much more fitting.
* Minor map changes, such as a extra oxygen cannister and similar.

Fixes https://github.com/tgstation/tgstation/issues/28947

:cl: Steelpoint (Ancient Station)
add: Major changes to Ancient Station include new sounds for the prototype RIG hardsuit, the NASA Engineering Voidsuit slowing the user down properly and new insulated gloves in engineering.
add: Several minor changes, such as an extra oxygen tank, spawn in equipment and minor tile changes.
/:cl:

### CODE REVIEW

**Comment by DoctoreMad** [comment]
>Slowdown of 4

Isn't that unnecessarily painful.

**Comment by Kraseo** [comment]
there's no slowdown in space

**Comment by Steelpoint** [comment]
Its to discourage them to wear it indoors, in space there's no movement penalty.

**Comment by Ergovisavi** [comment]
Consider that any slowdown on non armored gear is entirely irrelevant because they will just hold it in their hands, and that having the slowdown is just annoying for everyone involved.

I get the intent, but in practice all you're doing is being annoying.

**Comment by DoctoreMad** [comment]
Can you give botany some roundstart saltpetre(maybe) and pest spray(DEFINITELY)? Grapes contain sugar which increase pest levels, but sometimes its the only plant that is gaining potency.

**Comment by DoctoreMad** [comment]
Oh
I see

**Comment by PraiseRatvar** [comment]
> All station helmets and the hardsuit helmet no longer block vision.

not a fan of this nerf, like people said on your feedback thread, git gud

**Comment by Steelpoint** [comment]
I'll see how the general mood before I commit that change, I've still got other things to work on for this PR.

**Comment by Steelpoint** [comment]
I've removed the NT Banners as I did not realise they can be used to heal people, I've also made some minor spelling changes.

**Comment by DoctoreMad** [comment]
Very rude

**Comment by Repukan** [comment]
Don't nerf the helmets, they're cool like that.

Also the banners are hilarious.

**Comment by Steelpoint** [comment]
I'll likely keep the helmet visor limitation.

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -22,6 +22,21 @@
...
+	icon_state = "yellow"
+	item_state = "ygloves"
+	siemens_coefficient = 1
+	permeability_coefficient = 0.05
+	item_color="yellow"
+	resistance_flags = 0
+
+
+/obj/item/clothing/gloves/color/fyellow/New()
+	..()
```
Comment (Cyberboss) [inline code/modules/clothing/gloves/color.dm]: Initialize and . = ..()

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by juanototo2** [comment]
The banners are ridiculous but they are hilariously fun. Keep the "whacking people with toolboxes while waving around a flag that instantly closes wounds" meme. PLEASE.

**Comment by Jalleo** [comment]
Perhaps make it so the old helmets visors are toggleable? When up the helmet has half protection or something like that.

**Comment by PraiseRatvar** [comment]
also please add an airlock somewhere on delta or charlie so people don't have to make a giant gaping hole for mechs or borgs

**Comment by Armhulen** [comment]
WHY DO PEOPLE ENJOY WELDER VISION JESUS

**Comment by Repukan** [comment]
We don't, but if we add QoL changes then it follows more QoL will be added, thus killing the spirit of the ruin.

Slibbery slobe

**Comment by MimicFaux** [comment]
The entire point of the ruin is to enjoy a hard-mode reclamation project, and so far 9/10 runs on it have been made fun by the difficulty level and roleplay (probably a bug) with other survivors. Everything from growing plants for healing, multiple trips to recharge shitty guns, to electrocuting myself on the solar arrays because someone else turned the grid on, have made the experience for me.

I like the vision reduction, I enjoyed the lack of gloves (but won't complain for budget gloves+), there was even something thematic about picking through the chemical storage beakers. What I would like to see is some more piece to the remains of atmos added for additional projects the spessmen can work on.

**Comment by Steelpoint** [comment]
PR is still a work in progress, I'll post again when my next set of changes and alterations is up.

**Comment by Really-Good-Soda-Flavor** [comment]
These look like stupid, unnecessary changes that nobody asked for to me.

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 2]
```diff
@@ -5204,6 +5208,46 @@
...
+"lu" = (
+/obj/item/weapon/stock_parts/manipulator,
+/turf/open/floor/plasteel/airless/floorgrime,
+/area/template_noop)
+"lv" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/item/weapon/stock_parts/console_screen,
+/turf/open/floor/plasteel/white/side{
+	tag = "icon-whitehall (EAST)";
+	icon_state = "whitehall";
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: dirty vars
(you can just remove these two lines with notepad or w/e tbh)

File: code/modules/clothing/gloves/color.dm (hunk 1 above)
Comment (Steelpoint) [inline code/modules/clothing/gloves/color.dm]: Don't know what your asking me to change as its identical to the other similar code entry for budget gloves.

**Comment by Steelpoint** [comment]
New push has some changes. Helmets and visors have the tinting again, I've removed the airless tiles on Charlie Station since I think that just raises unnecessary issues with broken atmo, and the fact its a non-issue to resolve. I've also scattered the parts to construct a sleeper, if you can find the parts.

Also some other minor mapping changes.

**Comment by Repukan** [comment]
Don't add parts to build a sleeper, they're already available if you deconstruct other machines.

**Comment by Armhulen** [comment]
at least not the hardsuit having tint aw god compromise

**Comment by Tacolizard** [comment]
the voltage in the solars made it so that cutting or laying down any wire connected to them would shock you. It is better that now we get some insulation, but it's dumb to basically force the player to shock themselves in order to wire the solars.

**Comment by Tacolizard** [comment]
also this map shouldn't be called oldstation (like it is in the filename) because that's the name of the original pre-boxstation map.

**Comment by PraiseRatvar** [comment]
just it call it steelstation

**Comment by Steelpoint** [comment]
I should mention I have removed the floor tiles on the solar array.

**Comment by Ergovisavi** [comment]
> I've removed the NT Banners as I did not realise they can be used to heal people, I've also made some minor spelling changes.

Keep them in but change the effect to only work if you're below 20 HP and not in crit or something. It boosts morale, but that'll only keep you trucking along, it won't completely heal you.

You keep some of the absurdity for fun, but keep it from turning you into fucking wolverine.

**Comment by DoctoreMad** [comment]
Why do the transit tubes have huge kPa anyways?

**Comment by Steelpoint** [comment]
I got some further changes I'm making.

I might add blueprints to help in construction.

**Comment by Steelpoint** [comment]
Pushed some changes, so my PR is now merge ready.

* The three spawn roles now have some extra equipment on them to differentiate them a bit. Security Officer spawns with a flash and handcuffs (plus his uniform having 10 melee makes him more combat focused). Engineer spawns with a emergency oxygen tank and a worn our insulated glove. Scientist spawns with a bandage in their pocket.
* Removed the tinting on the hardsuit helmet, I think that's fair considering its slow down.
* Reworked minor tiles, such as the solar arrays. Also got rid of the excess items to reconstruct a sleeper without raiding other machines.

**Comment by Steelpoint** [comment]
Also @DoctoreMad the transit tubes had high pressure due to the lack of air on Charlie Station seeping into the tubes.

My removal of the airless tiles on Charlie should stop this issue.

File: code/modules/clothing/gloves/color.dm [hunk 3]
```diff
@@ -22,6 +22,20 @@
...
+	desc = "Old and worn out insulated gloves, hopefully they still work."
+	name = "worn out insulated gloves"
+	icon_state = "yellow"
+	item_state = "ygloves"
+	siemens_coefficient = 1
+	permeability_coefficient = 0.05
+	item_color="yellow"
+	resistance_flags = 0
+
+/obj/item/clothing/gloves/color/old/New()
```
Comment (AnturK) [inline code/modules/clothing/gloves/color.dm]: Initialize, also this should just be subtype of fake yellows.

Comment (Steelpoint) [inline code/modules/clothing/gloves/color.dm]: What does initialize mean?

Comment (AnturK) [inline code/modules/clothing/gloves/color.dm]: It's a function we call on objects immediately once map is fully loaded and ready. Replace New with Initialize, we have more control over initialization times and process with it. See https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/atoms.dm if you want more details.

**Comment by Steelpoint** [comment]
Asked for changes have been made, PR is merge ready assuming nothing breaks.

**Comment by lordpidey** [comment]
So... how does an area with LOW pressure seeping into the tube cause HIGH pressure?  That seems like it's a bug in it's own right.

**Comment by Steelpoint** [comment]
This is still good to merge. I addressed the requested changes as @Cyberboss requested.

**Comment by octareenroon91** [comment]
How many oxygen tanks are there overall?

**Comment by Steelpoint** [comment]
Two I believe.

**Comment by Steelpoint** [comment]
So, err, is this going to get merged or anything anytime soon?

@Cyberboss for approving the changes I made.

File: code/modules/clothing/gloves/color.dm [hunk 4]
```diff
@@ -22,6 +22,14 @@
 	..()
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
+/obj/item/clothing/gloves/color/fyellow/old
+	desc = "Old and worn out insulated gloves, hopefully they still work."
+	name = "worn out insulated gloves"
+
+/obj/item/clothing/gloves/color/fyellow.old/Initialize()
+	..()
```
Comment (Cyberboss) [inline code/modules/clothing/gloves/color.dm]: . = ..()

File: code/modules/clothing/gloves/color.dm [hunk 5]
```diff
@@ -22,6 +22,14 @@
 	..()
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
+/obj/item/clothing/gloves/color/fyellow/old
+	desc = "Old and worn out insulated gloves, hopefully they still work."
+	name = "worn out insulated gloves"
+
+/obj/item/clothing/gloves/color/fyellow.old/Initialize()
```
Comment (Cyberboss) [inline code/modules/clothing/gloves/color.dm]: /

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Steelpoint** [comment]
Fixes have been deployed.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**